extensions/amp-access-poool/0.1/poool-impl.js

```diff
@@ -175,7 +175,7 @@ export class PooolVendor {
    * @private
    */
   renderPoool_() {
-    const pooolContainer = document.getElementById('poool');
+    const pooolContainer = this.ampdoc.win.document.getElementById('poool');
```

Comment:
This should actually be `this.ampdoc.getElementById(...)`

Reply:
Is it better to stick with `this.ampdoc.getElementById(...)` or move to `this.ampdoc.getRootNode().querySelector('#poool)` here?

Reply:
I have no preference: they both execute against `rootNode` and that's what we need here.

Reply:
Alright that was my next question, thanks for the info!